docs/release-notes/v11.5.3.md

```diff
@@ -0,0 +1,20 @@
+---
+id: v11.5.3
+title: Version v11.5.3 Release Notes
+sidebar_label: v11.5.3
+image: "img/release-note-link-preview.png"
+---
+
+_Oct 5th, 2023_
+
+## Console
+
+### Bug Fix
+
+#### Inability to create cluster from empty-state
```

Comment:
let's put a warning in 11.5.0 RN that has this bug and has been fixed in v11.5.3 with a link to this page

Reply:
Why? We never did that and I have never seen this practice in RNs. We should do the same also with v11.5.1 and v11.5.2, seems too much to me